<ui version="4.0" >
 <class>ImportWidget</class>
 <widget class="QWidget" name="ImportWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>811</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbGeneral" >
     <property name="title" >
      <string>Datafile</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="leFileName" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="pbBrowse" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Browse</string>
        </property>
        <property name="default" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="pbInfo" >
        <property name="text" >
         <string>Info</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QLabel" name="lFileType" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>577</width>
          <height>22</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="indent" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>File type :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="KComboBox" name="cbFileType" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <item>
         <property name="text" >
          <string>automatic</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" >
      <item row="2" column="4" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>End row :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="KIntNumInput" name="niStartRow" >
        <property name="value" >
         <number>1</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Start row :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" >
       <widget class="KIntNumInput" name="niEndRow" >
        <property name="minimum" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QCheckBox" name="cbImportHeader" >
        <property name="text" >
         <string>Import header</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2" >
       <widget class="QCheckBox" name="cbSimplifyWhitespaces" >
        <property name="text" >
         <string>Simplify whitespaces</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QCheckBox" name="cbEmptyLines" >
        <property name="text" >
         <string>Allow empty lines</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3" >
       <widget class="QComboBox" name="cbSeparatingCharacter" >
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="5" >
       <widget class="QComboBox" name="cbCommentCharacter" >
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLabel" name="lSeparatingCharacter" >
        <property name="text" >
         <string>Separating character</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4" >
       <widget class="QLabel" name="lCommentCharacter" >
        <property name="text" >
         <string>Comment character</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBinaryOptions" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Binary Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Format :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="cbBinaryFormat" />
      </item>
      <item row="3" column="2" >
       <widget class="QComboBox" name="cbBinaryByteOrder" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Byte order :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Fields :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="KIntNumInput" name="niBinaryFields" >
        <property name="value" >
         <number>2</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QCheckBox" name="cbCreateSpreadsheet" >
       <property name="text" >
        <string>Create new spreadsheet</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbUseFilename" >
       <property name="text" >
        <string>Use filename as spreadsheet title</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbBrowse</sender>
   <signal>clicked()</signal>
   <receiver>ImportWidget</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>632</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel" >
     <x>363</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbInfo</sender>
   <signal>clicked()</signal>
   <receiver>ImportWidget</receiver>
   <slot>fileInfoDialog()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>646</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel" >
     <x>438</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFileName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportWidget</receiver>
   <slot>updateFileType()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>242</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile()</slot>
  <slot>updateFileType()</slot>
  <slot>fileInfoDialog()</slot>
 </slots>
</ui>
